cms/templates/home_page.html

```diff
@@ -13,21 +13,11 @@
 {% render_bundle 'header' %}
 {% endblock %}
 
-{% block scripts %}
-  {{ block.super }}
-  <script type="text/javascript" src="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
-  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
-  <script src="{% static 'js/vendor/jquery.fancybox.min.js' %}"></script>
-  <script src="{% static 'js/hero.js' %}"></script>
-  <script src="{% static 'js/testimonials_carousel.js' %}"></script>
-  <script src="{% static 'js/courseware_carousel.js' %}"></script>
-  <script src="{% static 'js/text_video_section.js' %}"></script>
-  <script src="{% static 'js/image_carousel.js' %}"></script>
-{% endblock %}
-
 {% block extrahead %}
   {{ block.super }}
-  {# This is included separately because FancyBox isn't used anywhere else so putting into the styles bundle isn't optimal. #}
+  {# This is included separately because FancyBox isn't used anywhere else so putting into the styles bundle isn't optimal. 
+    Also fancybox does not supply scss and the css in the distribution/source does not play nice with webpack
+  #}
```

Comment:
This text is showing up in the ui:

![Screenshot_2020-01-21 Home](https://user-images.githubusercontent.com/28598/72822921-e83b5f00-3c40-11ea-86b2-9b7165ca65b2.png)